src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt

```diff
@@ -24,10 +30,15 @@ class TaskDeploymentService {
     private lateinit var activeUserTasks: ActiveUserTasks
     @Autowired
     private lateinit var taskDestroyService: TaskDestroyService
-
+    @Autowired
+    private lateinit var awsParsing: AwsParsing
     @Autowired
     private lateinit var configurationResolver: ConfigurationResolver
 
+    @Value("classpath:policyDocuments/jupyterBucketAccesssPolicy.json")
+    lateinit var jupyterBucketAccessDocument: Resource
+    private lateinit var jupyterBucketAccessRolePolicyString: String
```

Comment:
I think we can remove the class level strings from their relating resource objects now that we only need them in a single function